src/template.ts

```diff
@@ -107,9 +107,10 @@ canvas.show {
   width: 100%;
   height: 100%;
   position: absolute;
-  background-size: cover;
+  background-size: contain;
+  background-repeat: no-repeat;
   background-position: center;
-  background-color: var(--poster-color, #fff);
+  background-color: var(--poster-color, inherit);
```

Comment:
I don't think this is actually correct. `inherit` should just apply the `background-color` of the parent, which is most likely not set because the parent is a `<slot>`.

Reply:
Interesting; this fixed my glitch, but you're right, it also broke our lazy loading examples (by break I mean the background is the wrong color). I think some other fix is still necessary...

Reply:
Fixed.